ci/test.sh

```diff
@@ -3,6 +3,16 @@
 set -euo pipefail
 
 ci/accept-third-party-license.sh
+
+echo "Hard linking ProxyImageNameSubstitutor into JDBC and R2DBC..."
+pushd spring-data-jdbc/src/test/java/org/springframework/data
+ln -f ../../../../../../../spring-data-relational/src/test/java/org/springframework/data/ProxyImageNameSubstitutor.java .
```

Comment:
How about copying the Java file? Hard linking seems unusual

Reply:
Done